doc/schemas/list-sets/schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "sets": {
```

Comment:
I was wondering about this special nesting level (let me call it "envelope") -> for now, we don't really need it.
Yet, we had cases in past, when Rule was expecting configuration like `[foo, bar]` and we couldn't extend it, causing a BC break to introduce `{items => [foo, bar], otherOption: true}`. let's be future ready